battleship-client/src/main/java/com/epam/solid/nie/client/ui/Ship.java

```diff
@@ -24,7 +24,6 @@ void hit(Point2D point2D) {
     }
 
     boolean isAlive() {
-        //return !(battleShip.getShipProperties().isEmpty());
```

Comment:
Yes please. No need to keep commented out code in the repository. That's what the repo is for.